source/utils.d

```diff
@@ -387,13 +361,84 @@ body
                       url,
                       retryCount,
                       retryCount > 1 ? "times" : "time");
-            downloadFile (url, dest, retryCount - 1);
+            download (url, dest, retryCount - 1);
         } else {
             throw e;
         }
     }
 }
 
+/**
+ * Download or open `path` and return it as a string array.
+ *
+ * Params:
+ *      path = The path to access.
+ *
+ * Returns: The data if successful.
+ */
+string[] getFile (const string path, const uint retryCount = 5)
```

Comment:
This should probably rather be `getFileContent` because `getFile` makes me think I get a `File` or a filename back.